### PR TITLE
Show 20 as current and latest release

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Review by skjnldsv — CHANGES_REQUESTED** [review]
https://github.com/nextcloud/documentation/blob/a25a363592bb7479dee4c9225102ba7334aca053/conf.py#L39-L40

File: conf.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 # building the versions list
 version_start = 17		# THIS IS THE SUPPORTED VERSION NUMBER
```
Comment (skjnldsv) [inline conf.py:39]: Well, I assume 17 is EOL now then?

Comment (rullzer) [reply conf.py:39]: Thursday. So kinda yes

Comment (ChristophWurst) [reply conf.py:39]: Could you kinda approve the PR in the meantime to get the docs landing page updated for 20?

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**